client/src/common/components/Comments/NewComment.jsx

```diff
@@ -28,38 +40,60 @@ class NewComment extends PureComponent {
     }
   };
 
-  handleAddNewComment = () => {
-    const { onAddNewComment } = this.props;
+  handleAddComment = () => {
+    const { onAddComment } = this.props;
     const { comment } = this.state;
 
-    onAddNewComment(comment);
+    if (!_isEmpty(_trim(comment))) {
```

Comment:
save this as a constant
const commentNotEmpty = !_isEmpty(_trim(comment));
and use inside this if

Reply:
I save only `_trim(comment)` because it is used twice